### DISCUSSION TITLE
[Question]: I hope for everyone's support, as I'm having an issue with images not being displayed when DALL-E returns the results.

### DISCUSSION THREAD

**Discussion by letiennam29** [danny-avila/LibreChat#1112] [Troubleshooting]
### What is your question?

Image display error.

### More Details

I hope for everyone's support, as I'm having an issue with images not being displayed when DALL-E returns the results.

![image](https://github.com/danny-avila/LibreChat/assets/5987534/8ff69ed9-decc-4eb0-961e-44379190101a)

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/5987534/ae883955-8232-4812-afd7-a23c074c4cfb)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
It really depends on how the LLM decides to answer. Try improving the prompt.

![image](https://github.com/danny-avila/LibreChat/assets/110412045/fbd1155f-6e3f-4347-9e48-d1e69a7731d0)

**Comment by letiennam29** [comment]
Thank you, I will try out more scenarios.
